README.md

```diff
@@ -709,7 +709,7 @@ z.string().regex(regex);
 z.string().includes(string);
 z.string().startsWith(string);
 z.string().endsWith(string);
-z.string().datetime(); // ISO 8601; default is without UTC offset, see below for options
+z.string().datetime(); // ISO 8601; default is UTC with zero offset, see below for options
```

Comment:
Choosing more explicit wording here.

"Without UTC offset" is potentially misleading, as there is some nuance in the fact that "Z" is a zone indicator for the zero UTC offset as opposed to the absence of an offset.